h/views/api/groups.py

```diff
@@ -177,7 +178,6 @@ def upsert(context, request):
 
     group = group_update_service.update(group, **update_properties)
 
-    # Note that this view takes a ``GroupUpsertContext`` but uses a ``GroupContext`` here
```

Comment:
This much be an outdated comment, it doesn't appear to make any sense and there is no `GroupUpsertContext`